docs/resource/process.md

```diff
@@ -27,32 +28,40 @@
+| [`process.command_line`](../attributes-registry/process.md) | string | The full command used to launch the process as a single string representing the full command. On Windows, can be set to the result of `GetCommandLineW`. Do not set this if you have to assemble it just for monitoring; use `process.command_args` instead. | `C:\cmd\otecol --config="my directory\config.yaml"` | `Conditionally Required` [3] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`process.executable.name`](../attributes-registry/process.md) | string | The name of the process executable. On Linux based systems, can be set to the `Name` in `proc/[pid]/status`. On Windows, can be set to the base name of `GetProcessImageFileNameW`. | `otelcol` | `Conditionally Required` [4] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`process.executable.path`](../attributes-registry/process.md) | string | The full path to the process executable. On Linux based systems, can be set to the target of `proc/[pid]/exe`. On Windows, can be set to the result of `GetProcessImageFileNameW`. | `/usr/bin/cmd/otelcol` | `Conditionally Required` [5] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.owner`](../attributes-registry/process.md) | string | The username of the user that owns the process. | `root` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.parent_pid`](../attributes-registry/process.md) | int | Parent Process identifier (PPID). | `111` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.pid`](../attributes-registry/process.md) | int | Process identifier (PID). | `1234` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**Additional attribute requirements:** At least one of the following sets of attributes is required:
+**[1]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[2]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[3]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[4]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[5]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+<!-- endsemconv -->
+
+### Selecting process attributes
+
+Instrumentations MUST populate at least one of the following attributes:
```

Comment:
Does this MUST apply generally (like for example `service.name`), or only when a resource detector for processes is explicitly activated?

I assume it's the latter, but the strong MUST is a bit misleading.

Reply:
I returned the non-normative phrase (`At least one of the following attributes is required`) to avoid strong MUST.

But yes, req level is only applied within the semconv (i.e. if you follow the semconv at all, you MUST populate required attributes)